configure_infrastructure.py

```diff
@@ -0,0 +1,115 @@
+"""
+...
+"""
+import os
+import sys
+from typing import (
+    List,
+)
+
+DOCKER_WAIT = 5
+DOCKERFILE_DATASTORES = [
+    "postgres",
+    "mysql",
+    "mongodb",
+    "mssql",
+]
+EXTERNAL_DATASTORES = [
+    "snowflake",
+    "redshift",
+]
+IMAGE_NAME = "fidesops"
+
+
+def configure_infrastructure(
```

Comment:
Just a thought- since this file can optional run tests and other things (on top of configure infra), is there a more appropriate name for this file?

Reply:
There likely is, perhaps something like `dispatch_commands` is more appropriate but I couldn't think of anything obvious that made sense so I left it. Open to suggestions of course

Reply:
Something like`datastore_infrastructure_dispatcher.py` or `command_dispatcher.py` or `infrastructure_command_dispatcher.py` comes to mind. Something that denotes we're running commands on datastores/infra, not just setting up the env.

Reply:
Have renamed to `run_infrastructure`